token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -398,6 +398,23 @@ There are **Primitive Rules** and **Composed Rules** that are created by combini
 
 More details of the Token Authorization Rules program, including examples, can be found [here](https://github.com/metaplex-foundation/mpl-token-auth-rules/blob/main/README.md).
 
+### Token Metadata Operations subject to Authorization Rules
```

Comment:
It's probably worth adding a brief blurb about the schema we're using.
<Operation:Scenario>